### PR TITLE
Update hip-226 response JSON

### PR DESCRIPTION
**Pull request by Nana-EC**
Signed-off-by: Nana-EC <nana.essilfie-conduah@hedera.com>

**Description**:
Update Contract Results response retrieved via timestamp `GET /api/v1/contracts/{id}/results/{timestamp}` and TransactionId `GET /api/v1/contracts/results/{transactionId}` with missing values

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: HIP/hip-226.md [hunk 1]
```diff
@@ -87,13 +89,16 @@ The following JSON represents a typical response result from either of these cal
...
       "topics": [
         "0xaf846d22986843e3d25981b94ce181adc556b334ccfdd8225762d7f709841df0",
         "0x0000000000000000000000000000000000000000000000000000000000000765"
       ]
     }
   ],
+  "result": "SUCCESS",
   "state_changes": [
     {
       "after": "0xaf846d22986843e3d25981b94ce181adc556b334ccfdd8225762d7f709841df0",
```
Comment (steven-sheehy) [inline HIP/hip-226.md:104]: Update state_changes to reflect latest design.

File: HIP/hip-226.md [hunk 2]
```diff
@@ -78,7 +78,9 @@ The following JSON represents a typical response result from either of these cal
     {
```
Comment (steven-sheehy) [inline HIP/hip-226.md:78]: Please remove `child_transactions`

File: HIP/hip-226.md [hunk 3]
```diff
@@ -124,7 +130,7 @@ These additional API endpoints do not alter exsiting REST API functionality.
 - Hedera Mirror Node design document
 - Description and code examples of queries added to Hedera REST API documentation section
 - The OpenAPI spec at `api/v1/docs` should be updated to reflect the new endpoints and allow users to test out the calls.
-- Reviewed in Engineeing Insights
+- Reviewed in Engineering Insights
```
Comment (steven-sheehy) [inline HIP/hip-226.md:135]: We should add what query parameters are supported

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: HIP/hip-226.md (hunk 1 above)
Comment (Nana-EC) [reply HIP/hip-226.md]: Updated

File: HIP/hip-226.md (hunk 2 above)
Comment (Nana-EC) [reply HIP/hip-226.md:77]: Removed

File: HIP/hip-226.md (hunk 3 above)
Comment (Nana-EC) [reply HIP/hip-226.md:138]: Added

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Pull request merged**